src/applications/yellow-ribbon/sass/yellow-ribbon.scss

```diff
@@ -0,0 +1,9 @@
+@media (min-width: 768px) {
+  .school-program {
+    margin-top: 36px !important;
+  }
+}
+
+.vads-u-outline--none:focus {
+  outline: none;
+}
```

Comment:
I think we may want to just use `va` instead of `vads` unless we can make this an official design system utility. Sorry, should have mentioned that during our call